qc_grader/grader/common.py

```diff
@@ -295,6 +302,8 @@ def serialize_answer(answer: Any, **kwargs: bool) -> Optional[str]:
         payload = pauliop_to_json(answer)
     elif isinstance(answer, (PulseQobj, QasmQobj)):
         payload = qobj_to_json(answer)
+    elif isinstance(answer, SamplerResult):
+        payload = SamplerResult_to_json(answer)
```

Comment:
will the reverse (deserialization) get added to the server?

Reply:
I will. To be honest i forgot about it but I'll be working on it today